internal/cmd/asyncapi/command_import_test.go

```diff
@@ -0,0 +1,13 @@
+package asyncapi
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestResolveSchemaType(t *testing.T) {
+	require.Equal(t, resolveSchemaType("avro"), "AVRO")
+	require.Equal(t, resolveSchemaType("json"), "JSON")
+	require.Equal(t, resolveSchemaType("proto"), "PROTOBUF")
```

Comment:
Use `assert` when testing multiple cases in one function
```suggestion
	assert.Equal(t, resolveSchemaType("avro"), "AVRO")
	assert.Equal(t, resolveSchemaType("json"), "JSON")
	assert.Equal(t, resolveSchemaType("proto"), "PROTOBUF")
```